app/scripts/views/marketing_snippet.js

```diff
@@ -3,45 +3,89 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 /**
- * Handles the marketing snippet on the 'ready' page.
- *
- * Shows `Get Sync on Firefox for Android` for users that complete
- * signup for sync in Firefox Desktop.
+ * Handles the Firefox for Mobile marketing snippet. Shows an
+ * iOS iTunes logo if user completes on iOS, shows a Google Play store logo
+ * if completing on Android, shows both logos if completing on
+ * neither.
  */
 
 define(function (require, exports, module) {
   'use strict';
 
+  const _ = require('underscore');
   const BaseView = require('views/base');
   const Cocktail = require('cocktail');
   const Constants = require('lib/constants');
   const MarketingMixin = require('views/mixins/marketing-mixin');
   const Template = require('stache!templates/marketing_snippet');
   const VerificationReasonMixin = require('views/mixins/verification-reason-mixin');
 
-  var View = BaseView.extend({
+  const APP_STORE_BUTTON = 'apple_app_store_button';
+  const PLAY_STORE_BUTTON = 'google_play_store_button';
+
+  const FORMAT_SVG = 'svg';
+  const FORMAT_PNG = 'png';
+
+  const playStoreImageLanguages = [
+    'ca',
+    'cs',
+    'da',
+    'de',
+    'en',
+    'es',
+    'et',
+    'fr',
+    'hu',
+    'id',
+    'it',
+    'ja',
+    'ko',
+    'lt',
+    'nb-NO',
+    'nl',
+    'pl',
+    'pt-BR',
+    'pt',
+    'ru',
+    'sk',
+    'sl',
+    'sv',
+    'tr',
+    'uk',
+    'zh-CN',
+    'zh-TW'
+  ];
+
+  const View = BaseView.extend({
     template: Template,
 
-    initialize (options) {
-      options = options || {};
-
+    initialize (options = {}) {
+      this._language = options.language;
       this._service = options.service;
     },
 
     context () {
-      var shouldShowMarketing = this._shouldShowSignUpMarketing();
+      const shouldShowMarketing = this._shouldShowSignUpMarketing();
+      const isIos = this._isIos();
+      const isAndroid = this._isAndroid();
+      const playStoreImage = this._storeImage(PLAY_STORE_BUTTON, FORMAT_PNG);
+      const appStoreImage = this._storeImage(APP_STORE_BUTTON, FORMAT_SVG);
 
       return {
+        appStoreImage: appStoreImage,
+        isAndroid,
+        isIos,
+        isOther: ! isIos && ! isAndroid,
+        playStoreImage: playStoreImage,
```

Comment:
Do my eyes deceive me or can `appStoreImage` and `playStoreImage` use shorthand properties as well?